src/ontology/mondo-edit.obo

```diff
@@ -372482,13 +372482,14 @@ subset: rare
 synonym: "body of uterus squamous cell carcinoma" EXACT [MONDO:patterns/location]
 synonym: "endometrial squamous cell carcinoma" RELATED [Orphanet:213716]
 xref: GARD:20479 {source="MONDO:GARD"}
-xref: ICD10CM:C50 {source="MONDO:relatedTo", source="Orphanet:213716/ntbt", source="Orphanet:213716"}
+xref: ICD10CM:C54.1 {source="MONDO:relatedTo", source="Orphanet:213716/ntbt", source="Orphanet:213716"}
```

Comment:
@katiermullen did you mean to have this x-ref as equivalent? 
(I can update the PR)